runtime/compiler/optimizer/VectorAPIExpansion.hpp

```diff
@@ -172,10 +175,9 @@ class TR_VectorAPIExpansion : public TR::Optimization
    */
    struct methodTableEntry
       {
-      TR::Node * (* _methodHandler)(TR_VectorAPIExpansion *, TR::TreeTop *, TR::Node *, TR::DataType, vec_sz_t, handlerMode);
-      TR::DataType _elementType;  // for high level API methods only
-      vapiArgType  _returnType;
-      vapiArgType  _argumentTypes[10];
+      TR::Node * (* _methodHandler)(TR_VectorAPIExpansion *, TR::TreeTop *, TR::Node *, TR::DataType, TR::VectorLength, int32_t, handlerMode);
+      vapiObjType  _returnType;
+      vapiObjType  _argumentTypes[10];
```

Comment:
Will rename _maxArguments into _maxNumberArguments (to be consistent with _maxNumberOperands) and use it instead of [10] above.  All assertions are in place when accessing corresponding arrays.

Reply:
Done